README.md

```diff
@@ -25,16 +25,28 @@
   
 ## Install & Run AmadeusGPT🎻
 
-- AmadeusGPT is a Python package hosted on `pypi`. You can create a virtual env (conda, etc, see below*) or Docker and run:
-```python
-pip install 'amadeusgpt[streamlit]'
+- We prepare installation script install.sh. Make sure you modify the bash script to fit your own.
```

Comment:
why are you deleting the pypi package? it is on pypi ;)

Reply:
Because I realize that, at least when I was working on it, one can't install deeplabcut with pip install alone. It needs to use the conda yaml to first install hdf5 and many users don't understand why that's the case. So inside the installation script, I force users to create a conda env via conda yaml and then pip install.

The installation script

```
source /Users/shaokaiye/miniforge3/bin/activate

conda env create -f conda/amadesuGPT.yml

conda activate amadeusgpt

pip install -e .[streamlit]
```

In colab, because hdf5 is installed, then they will run into slightly different problem such as pytable vs. tables from deeplabcut. I overheard you guys might have solved it. Let me know if there is better way to solve it.